GetUserInfo/GetUserInfo/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+
+namespace GetUserInfo
+{
+    class Program
+    {
+
+        static string askQuestion(string question)
+        {
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+
+        static void Main(string[] args)
+        {
+            string UserFirstName;
+            string UserLastName;
+            string UserFullName;
+
+            do
+            {
+                UserFirstName = askQuestion("Hello! Whats your first name?");
+                Console.WriteLine("Nice to meet you, " + UserFirstName + "!");
+
+                UserLastName = askQuestion("What's your last name?");
+
+                UserFullName = askQuestion("So your full name is " + UserFirstName + " " + UserLastName + "?\n(y/n)");
```

Comment:
This variable could probably use a better name. It seems like it should be used to store the user's full name, but it's actually used to store a yes/no response